src/main/java/edu/kis/powp/command/FigureScriptAdapter.java

```diff
@@ -0,0 +1,18 @@
+package edu.kis.powp.command;
+
+import edu.kis.powp.IFigureScript;
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class FigureScriptAdapter implements DriverCommand{
+
+    private final IFigureScript script;
+
+    public FigureScriptAdapter(IFigureScript script) {
+        this.script = script;
+    }
+
+    @Override
+    public void execute(Job2dDriver driver) {
+        script.figureScript(driver);
```

Comment:
Nice hack, but the goal was to generate a list of commands.
The solution could be obtain, e.g. by modifying further IPlotterTest.java